api/services/ocean/spark/parameters/query/oceanSparkClusterShouldDeleteOcean.yaml

```diff
@@ -0,0 +1,7 @@
+in: query
+name: shouldDeleteOcean
```

Comment:
@thorsteinnth it looks like this query param is not yet used in wave-core. Should we remove it from the doc?

Reply:
yes, we should remove it as well

Reply:
Removed, thanks!